### ISSUE TITLE
Error to display the map in Syncked maps

### ISSUE THREAD

**Issue by volpatoo** [TiagoOlivoto/plimanshiny#10]
After computing the indexes clicking on "Compute the indexes!" botton, the map cannot be visualized in the Syncked maps.  Instead of that an error is displaying in the panel.

![image](https://github.com/TiagoOlivoto/plimanshiny/assets/44096217/ed440eba-d364-4ab3-b1f2-e8957a40f097)

```r
Warning: Error in <Anonymous>: unable to find an inherited method for function ‘mapView’ for signature ‘"SpatRaster"’
  110: stop
  109: <Anonymous>
  108: mapView
  107: mapview
  103: mapview::mapview
  102: mosaic_view
   99: renderUI [C:/temp_plimanshiny/git/plimanshiny/R/mod_indexes.R#193]
   98: func
   85: renderFunc
   84: output$indexes_1-indexsync
    3: runApp
    2: print.shiny.appobj
    1: <Anonymous>
```

**Comment by TiagoOlivoto** [comment]
This sounds weird.. There is no call of "mapView" in any place of the code.

**Comment by TiagoOlivoto** [comment]
Fixed after updating mapview package to 2.11.2
